frontend/src/components/Banner.tsx

```diff
@@ -130,6 +130,7 @@ type BannerCtaProps = {
    * Callback to run when the call-to-action is activated. Optional if `target` is provided.
    */
   onClick?: () => void;
+  className?: string;
```

Comment:
One thing I'd be wary about here is that moving styles outside of the `<BannerCta>` component makes it hard to see how future changes to it will break its use here. Wouldn't it be just as easy to just directly use a regular `<button>` instead?